libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -2742,12 +2742,15 @@ static void prvSOCKETS_Threadsafe_SameSocketDifferentTasks( Server_t xConn )
             {
                 xReturned = SOCKETS_Recv( ( Socket_t ) xSocket, ( char * ) pcReceivedString, xRecvLen, 0 );
 
-                TEST_ASSERT_GREATER_THAN_MESSAGE( 0, xReturned, "Error occurred receiving large message" );
+                TEST_ASSERT_GREATER_OR_EQUAL_MESSAGE( 0, xReturned, "Error occurred receiving large message" );
```

Comment:
If `xReturned < 0` it would be good to log the value of `xReturned` (you might be able to use `tcptestFAILUREPRINTF()` for this) before this line to aid in debugging – I

Reply:
Yes, thank you.
I added the requested line. :)